src/main/java/Main.java

```diff
@@ -0,0 +1,35 @@
+import game.Game;
+import numberBaseball.IntegerParser;
+import numberBaseball.Parser;
+import numberBaseball.ResultPrinter;
+import numberBaseball.Setting;
+import random.RandomGenerator;
+import random.RandomIntegerGenerator;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+
+        RandomGenerator<Integer> randomGenerator = new RandomIntegerGenerator();
+        Parser<Integer> parser = new IntegerParser();
+        Game<Integer> game = new Game<>(randomGenerator);
+        ResultPrinter printer = new ResultPrinter(Setting.MAX_NUMBER_OF_ELEMENTS);
+
+        while(true){
+            printer.printStartMessage();
+            String input = sc.next();
+            boolean result = printer.print(game.tryNew(parser.parse(input)));
+            if(result){
+                input = sc.next();
+                if("2".equals(input)) break;
+                if("1".equals(input)) {
```

Comment:
```suggestion
                if(MENU_STOP.equals(input)) break;
                if(MENU_START.equals(input)) {
```

코드 내에 리터럴 문자열은 되도록 상수를 통해 의미를 명확하게 전달할 수 있는 방법이 좋을 것 같습니다.

Reply:
Setting 클래스에 MENU_STOP, MENU_START 변수로 분리하였습니다.